README.md

```diff
@@ -10,12 +10,11 @@ Build status:
 [![appveyor][appveyor-img]](https://ci.appveyor.com/project/JuliaLang/julia/branch/master)
 
 Code coverage:
-[![coveralls][coveralls-img]](https://coveralls.io/r/JuliaLang/julia?branch=master)
+[![Coverage Status](https://coveralls.io/repos/github/JuliaLang/julia/badge.svg?branch=master)](https://coveralls.io/github/JuliaLang/julia?branch=master)
```

Comment:
With this new badge (https://s3.amazonaws.com/assets.coveralls.io/badges/coveralls_77.svg) there is no reference to coveralls then? Currently the badges show `coveralls` and `codecov` which seem nicer compared to to after this PR where we will have `coverage` and `codecov`.

Reply:
Good point! I don't disagree.